src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -28,14 +35,31 @@ public SwerveSubsystem(
     if (Utils.isSimulation()) {
       startSimThread();
     }
+
+    Translation2d[] kinematicsTranslations = new Translation2d[4];
+    for (int i = 0; i < 4; i++) {
+      kinematicsTranslations[i] = new Translation2d(modules[i].LocationX, modules[i].LocationY);
+    }
+    kinematics = new SwerveDriveKinematics(kinematicsTranslations);
+
+    AutoBuilder.configureHolonomic(
+        () -> super.getState().Pose,
+        (a) -> super.seedFieldRelative(a),
+        () -> getChassisSpeeds(),
+        (a) ->
+            this.setControl(
+                new SwerveRequest.FieldCentric()
```

Comment:
```suggestion
                new SwerveRequest.RobotCentric()
```
This should be robot centric, as per pathplanner docs
Might solve a few issues